python/servo/build_commands.py

```diff
@@ -701,6 +697,11 @@ def package_generated_shared_libraries(libs, build_path, servo_exe_dir):
                 if not package_msvc_dlls(servo_exe_dir, target_triple, vcinstalldir, vs_version):
                     status = 1
 
+                # UWP build hololens
+                if uwp:
+                    if build_uwp_hololens(target_triple, dev):
```

Comment:
I was mistaken earlier - check_call will automatically cause the build to fail if it encounters a problem, so we don't need to check the return value and set status to 1.